src/imp/tracer_imp.js

```diff
@@ -1133,6 +1184,12 @@ export default class Tracer extends opentracing.Tracer {
                     if (res.errors && res.errors.length > 0) {
                         this._warn('Errors in report', res.errors);
                     }
+
+                    if (res.commandsList && res.commandsList.length > 0) {
+                        if (res.commandsList[0].devMode) {
```

Comment:
I wonder if we're really supposed to iterate over the commands? Or a TODO?

Reply:
I don't believe so - the command list is an array with one item, and all the commands are keys on that object.